docs/runbooks/CDIMultipleDefaultVirtStorageClasses.md

```diff
@@ -0,0 +1,32 @@
+# CDIMultipleDefaultVirtStorageClasses
+<!-- Edited by akalenyu, 10 Oct 2023 -->
+
+## Meaning
+
+The default virtualization storage class takes precedence over k8s default storage class for VM disk image creation purposes.
+
+This alert fires when there are multiple default virtualization storage classes.
+
+## Impact
+
+With multiple default storage classes present, any DataVolume requesting the default storage class (not explicitly specifying a storage class name) gets the most recently created default storage class, regardless of the default status of that storage class, and the administrator receives an alert.
+
+## Diagnosis
+
+Observe multiple storage classes are marked as default for virtualization purposes:
```

Comment:
Suggestion:

Obtain a list of default virtualization storage classes by running the following command: